### ISSUE TITLE
Skip the "paste here" button if there is only one parent

### ISSUE THREAD

**Issue by havutcuoglu** [contao/contao#2826]
Hi People, is it possible to generate or duplicate the frontend modules without "Paste Here" option, like by news archives?

This option make no sense because you can not add the module somewhere else. There is only one option.

It will make the usability easier and will save time.

![Bildschirmfoto 2021-03-03 um 09 13 24](https://user-images.githubusercontent.com/11348417/109776240-aaaa5300-7c02-11eb-9a62-7021c8f4ad10.png)

![Bildschirmfoto 2021-03-03 um 09 17 43](https://user-images.githubusercontent.com/11348417/109775989-6cad2f00-7c02-11eb-9eda-daca1ef5de8c.png)

**Comment by fritzmg** [comment]
You mean if there is only one theme?

**Comment by havutcuoglu** [comment]
If i have more then one theme, i have to choose the fe module. See attached image.
![Bildschirmfoto 2021-03-03 um 09 44 10](https://user-images.githubusercontent.com/11348417/109778783-8439e700-7c05-11eb-9257-e25ec4ce2e34.png)

It makes no sense is there one or more themes. Sorry but I can't understand your question.

**Comment by aschempp** [comment]
> you can not add the module somewhere else. There is only one option.

The idea is that you can add it to another theme. But we should think about if it would be better like news.

**Comment by havutcuoglu** [comment]
> The idea is that you can add it to another theme. But we should think about if it would be better like news.

Thanks. 👍

If I want to insert in a different theme, then I am in the wrong theme and can go back.

As I said, it's okay if there are only 10 or 20 modules, but over 50 and more you get crazy because you have to scroll up every time. This will make it easy to use.

**Comment by fritzmg** [comment]
> It makes no sense is there one or more themes. 

Why would that not make any sense? 

> If I want to insert in a different theme, then I am in the wrong theme and can go back.

You need to navigate to where ever you want to insert the module. That's how it works everywhere else in Contao too.

**Comment by havutcuoglu** [comment]
> Why would that not make any sense?

Usually for a website we need only one theme. I am working now over 8 years with Contao and one time I use two themes. And I create more than 300 websites with contao.

**Comment by fritzmg** [comment]
> Usually for a website we need only one theme.

Yes, that's why I was asking. So if only one theme is present, the _Paste here_ functionality could be omitted.

**Comment by havutcuoglu** [comment]
That means, if more than one theme is available, that "paste here" or "insert here" is integrated?

That could of course be useful.

**Comment by leofeyer** [comment]
As discussed in Mumble on April 8th, this is possible under the following conditions:

* there is a parent-child-relation and there is only one parent;
* there is no `sorting` field (e. g. when duplicating articles);
* there is no dynamic ptable.

**Comment by zoglo** [comment]
See #10103
